### PR TITLE
LPS 104982

### PR DESCRIPTION
**Pull request by mateomustapic**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/index.js [hunk 1]
```diff
@@ -18,6 +18,11 @@ const DEFAULT_CONFIG = {
 	toolbarId: 'pageEditorToolbar'
 };
 
+const INITIAL_STATE = {
+	discardDraftRedirectURL: '',
+	discardDraftURL: ''
```
Comment (julien) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/index.js]: This isn't needed, you're already injecting that from the Java display context

Comment (mateomustapic) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/index.js]: Yes, it is injected from Java display context, but so was previously in the old component and discardDraftRedirectURL and discardDraftURL had initial values in the state https://github.com/julien/liferay-portal/blob/1aab1cd79c35bef0781f5f2095c6928243d0be16/modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js#L242  , so I was following this.

Comment (julien) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/index.js]: I understand you'd want to know how it was done before, but you probably have noticed that now we're using React and things are a bit different. 

Have you noticed that all the other properties defined in the Java display context aren't defined in `INITIAL_STATE`, but do end up in the `ConfigContext`?

Comment (mateomustapic) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/index.js]: yes, correct. I will then remove these properties.

**Comment by julien** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/82270
Thanks!

**Pull request closed without merging**